ibtk/src/utilities/LibMeshSystemIBVectors.cpp

```diff
@@ -0,0 +1,108 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (c) 2020 - 2020 by the IBAMR developers
+// All rights reserved.
+//
+// This file is part of IBAMR.
+//
+// IBAMR is free software and is distributed under the 3-clause BSD
+// license. The full text of the license can be found in the file
+// COPYRIGHT at the top level directory of IBAMR.
+//
+// ---------------------------------------------------------------------
+
+/////////////////////////////// INCLUDES /////////////////////////////////////
+
+#include <ibtk/LibMeshSystemIBVectors.h>
+
+#include <tbox/Utilities.h>
+
+#include <libmesh/equation_systems.h>
+#include <libmesh/petsc_vector.h>
+#include <libmesh/system.h>
+
+#include <map>
+#include <memory>
+#include <string>
+#include <vector>
+
+/////////////////////////////// STATIC ///////////////////////////////////////
+
+/////////////////////////////// NAMESPACE ////////////////////////////////////
+
+namespace IBTK
+{
+/////////////////////////////// CLASS DEFINITION /////////////////////////////
+
+LibMeshSystemIBVectors::LibMeshSystemIBVectors(const std::vector<IBTK::FEDataManager*>& fe_data_managers,
+                                               std::string system_name)
+    : LibMeshSystemVectors(std::vector<bool>(fe_data_managers.size(), true), system_name),
```

Comment:
This is a bit risky because the parent class is going to be set up in some half-completed state. Lets move the rest of the code in the constructor into a convenience function that converts `std::vector<FEDataManager *>` to `std::vector<EquationSystems *>`.